src/pages/Host/Schedule/HostScheduleView.css

```
@@ -155,15 +173,19 @@
 }
 
 /* Timeline */
-.host-sv-timeline { display: flex; flex-direction: column; gap: 20px; }
+.host-sv-timeline {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
 
-.host-sv-day-group { }
+.host-sv-day-group {}
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)